packages/dashmate/docker-compose.platform.yml

```
@@ -6,6 +6,7 @@ services:
     restart: unless-stopped
     volumes:
       - drive_abci_data:/var/lib/dash/rs-drive-abci
+        drive_abci_logs:/var/log/dash
```

Shouldn't it be like this?
```suggestion
      - drive_abci_logs:/var/log/dash
```

---

yeah, we can do that